sda/status.go

```diff
@@ -0,0 +1,36 @@
+package sda
+
+//Status is the status server
+type Status map[string]string
```

Comment:
needs better description:
`// Status holds key/value pairs of the status to be returned to the requester`